pages/deployment/multicontainer.md

```diff
@@ -0,0 +1,80 @@
+
+Unlike single container applications, multicontainer applications do not run containers in privileged mode by default. If you want to make use of hardware, you will either have to set some services to privileged, using `privileged: true`, or use the `cap_add` and `devices` settings to map in the correct hardware access to the container.
+
+Here, the `gpio` service is set up to use **i2c** sensors:
+
+```
+gpio:
+    build: ./gpio
+    devices:
+      - "/dev/i2c-1:/dev/i2c-1"
+      - "/dev/mem:/dev/mem"
+    cap_add: 
+      - SYS_RAWIO
+```
+
+## resin.io settings
+
+There are a few settings and considerations specific to resin.io that need to be taken into account when building multicontainer applications. First, using the `INITSYSTEM=on` setting in the Dockerfile of a service is only supported if the container is run as privileged, as **systemd** does not run correctly in unprivileged containers.
+
+In addition, there are some resin.io specific labels that can be defined in the `docker-compose.yml` file. These provide access to certain bind mounts and environment variables without requiring you to run the container as privileged.
+
+Label | Default | Description
+--- | --- | ---
+io.resin.features.dbus | false | Bind mounts the hostOS dbus into the container using “/run/dbus:/host/run/dbus”
+io.resin.features.kernel_modules | false | Bind mounts the hostOS /lib/modules into the container. (i.e. “/lib/modules:/lib/modules”)
+io.resin.features.firmware | false | Bind mounts the hostOS /lib/firmware into the container
+io.resin.features.supervisor_api | false | Ensures that RESIN_SUPERVISOR_HOST, RESIN_SUPERVISOR_PORT, RESIN_SUPERVISOR_ADDRESS, and RESIN_SUPERVISOR_API_KEY are added to the container environment variables, so the supervisor API can be used. (Currently will only work for services that have network_mode = “host” or “bridge” )
+io.resin.features.resin_api | false | When enabled, it will make sure that RESIN_API_KEY is added to the container env vars
+
+These labels are applied to a specific service with the `labels:` setting
+
+```
+labels:
+      io.resin.features.dbus: "true"
+      io.resin.features.kernel-modules: "true"
+      io.resin.features.firmware: "true"
+      io.resin.features.supervisor-api: "true"
+```
+
+## Single container applications 
+
+If you push a project with only a Dockerfile or Dockerfile.template, rather than a `docker-compose.yml`, a single container will be built and downloaded, and the experience will be very similar to using resin.io before multicontainer support. In this form the single container, now known as a service, will be automatically assigned the name `main`, and you will see this service in a list on the device dashboard. This service will have host networking, be privileged, and have `lib/modules`, `/lib/firmware`, and `/run/dbus` bind mounted into the container automatically. 
```

Comment:
We should perhaps show what the auto-generated docker-compose.yml for that single service would look like, just so people have an idea